data/test/scenarios/test_remove_object_compatibility.cfg

```diff
@@ -0,0 +1,118 @@
+# This unit test is used for verify backward compatibility of remove_ability
+
+{GENERIC_UNIT_TEST "test_remove_object_compatibility" (
```

Comment:
This (and the filename) should probably say "remove_ability" rather than "remove_object".